### ISSUE TITLE
Lighting is broken on Linux with and AMD gpu

### ISSUE THREAD

**Issue by rgba16f** [o3de/o3de#11225]
**Describe the bug**
A large block of hte screen doesn't receive direct lighting

**Assets required**
AutomatedTesting/Levels/AtomFeatureIntegrationBenchmark

**Steps to reproduce**
1. On Linux open the editor
2. Load AtomFeatureIntegrationBenchmark level
3. See error

**Expected behavior**
Level lighting matches Windows DX12

**Actual behavior**
The lower portion of the screen has incorrect lighting, the portion changes with camera orientation and window size.

**Screenshots/Video**
![Screenshot from 2022-08-11 15-58-38](https://user-images.githubusercontent.com/82187279/184217822-389205b8-0858-4cad-bb20-83d01d8895f6.png)

**Found in Branch**
o3de development@2aac761cc65ad86a9e77ca54435ad5d7f1555a2f

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows 10
 - Version 20H2 19042.1826
 - CPU Intel(R) Core(TM) i7-6850K
 - GPU AMD Radeon RX Vega 64
 - Memory 32GB

**Comment by pollend** [comment]
@rgba16f can you try this test with this branch: https://github.com/o3de/o3de/pull/10876

**Comment by rgba16f** [comment]
> for the world if you attach the SSAO shader and disable it does the black boxes disappear?
> 
> seems similar to this issue: #4379
> 
> @rgba16f can you try this test with this PR: #10876

Huzzah @pollend, I can confirm that your PR does fix the lighting issue I'm seeing.

**Comment by pollend** [comment]
@rgba16f I'm seeing the same issue again can you re-verify this with development?

https://github.com/o3de/o3de/pull/11381

**Comment by rgba16f** [comment]
@pollend I'm seeing this issue on AMD as well

**Comment by pollend** [comment]
We need to tweak the barriers on that call. Pipeline barrier is getting ignored when we try to set it

**Comment by pollend** [comment]
@rgba16f did you verify this with the recent change that got in: https://github.com/o3de/o3de/pull/12764

**Comment by rgba16f** [comment]
> @rgba16f did you verify this with the recent change that got in: #12764

I didn't

**Comment by gralco** [comment]
I built 2210 on Fedora 37 and have this issue (I did not experience this on Fedora 36). All textures are shown as black. This does not seem to be a lighting issue.

I will try building and testing the head of development (92af95a) to see if the problem persists

**Comment by gralco** [comment]
@pollend #12764 fixes the issue for me on Fedora 37.

For me, the issue was that all textures in the viewport and in the editor's thumbnail previews showed up black, so I'm not certain if it's identical to what @rgba16f was experiencing given the screenshots.

**Comment by codingchipmunk** [comment]
> #12764

I had the same issue with 22.10 on Linux with an AMD GPU and Mesa drivers. I just built the engine from the current `development` branch and can confirm that it is solved.

My issue was the very same as described by  @rgba16f, down to the point where I saw the same pattern of the black area shown in the original screenshot. In addition, I had the problem where resizing the viewport too fast or often lead to an access violation in Mesa and a complete system hang which required a reset. But that issue is also solved now.
